atomic_reactor/schemas/config.json

```diff
@@ -723,6 +723,14 @@
               "required": ["old", "new"],
               "additionalProperties": false
             }
+          },
+          "skip_all_allow_list": {
+            "description": "Koji packages allowed to use skip_all option in container.yaml",
+            "type": ["array", "null"],
+            "items": {
```

Comment:
It should be property with package name and reason why it is allowed.

Reply:
we will have only list, and comment for each package with that Vault Exception ID

Reply:
You cannot enforce that using comment, but ok